CHANGELOG.md

```diff
@@ -18,6 +18,8 @@ and includes an additional section for migration notes.
 
 ### Changed
 - *ORCA-478* Updated bucket policy documentation for deep glacier bucket in DR account so that the users now can only upload objects with storage type as either `GLACIER` or `DEEP_ARCHIVE`.
+- *ORCA-457* `RequestFiles` will now raise a descriptive error when user attempts to recover `DEEP_ARCHIVE` files with the `Expedited` recovery method.
```

Comment:
should also add more info on this on user documentation. Not sure which doc to update though for recovery